internal/pkg/config/loader.go

```diff
@@ -41,13 +44,17 @@ func LoadFromFile(profile string, configuration interface{}) error {
 	}
 	contents, err := ioutil.ReadFile(fileName)
 	if err != nil {
-		return fmt.Errorf("could not load configuration file (%s): %v", fileName, err.Error())
+		msg := fmt.Sprintf("could not load configuration file (%s): %s", fileName, err.Error())
```

Comment:
You don't need to explictly call `err.Error()` here, go will just call the Error method for you. It's easier to read if you leave out the `Error()` method.